dts/bindings/sensor/hzgrow,r502a.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2021, Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+description: HZ-Grow GROW_R502A Fingerprint sensor.
+
+compatible: "hzgrow,r502a"
+
+include: uart-device.yaml
+
+properties:
+    address:
+      type: int
+      required: false
+      default: 0xffffffff
```

Comment:
Even though uart isn't strictly a bus, it can be depending on the physical configuration. In Zephyr, `uart-controller.yaml` defines UART as a bus, so you can add devices connected to the UART as child nodes and use the standard `reg` property.

Reply:
understood and fixed.